components/LatestCode.js

```diff
@@ -1,16 +1,10 @@
-import React, { useEffect, useState } from 'react';
-import Link from 'next/link';
-import getLatestRepos from '@lib/getLatestRepos';
-import userData from '@constants/data';
+import userData from "@constants/data";
 
-export default function LatestCode({ repositories }) {
-  const [repos, setRepos] = useState([]);
+export default function LatestCode({ repositories = [] }) {
+  if (repositories.length === 0) {
+    return null;
+  }
```

Comment:
**Question:** Not sure if this is the appropriate place to put this check. Might be better in the parent component?